src/features/unified-share-modal/UnifiedShareModal.js

```diff
@@ -367,16 +371,28 @@ class UnifiedShareModal extends React.Component<Props, State> {
         this.setState({ isEmailLinkSectionExpanded: false });
     };
 
+    hasExternalContact = (contacts: Array<Contact>) => {
+        return contacts.some(contact => contact.isExternalUser);
```

Comment:
Please verify this works as expected in IE11

Reply:
Good catch. I just verified on my Windows VM in IE11, and it works.